packages/twenty-front/src/modules/workflow/components/WorkflowEditActionFormServerlessFunction.tsx

```diff
@@ -63,12 +89,26 @@ export const WorkflowEditActionFormServerlessFunction = (
     1_000,
   );
 
-  const handleInputChange = (key: string, value: any) => {
-    const newFunctionInput = { ...functionInput, [key]: value };
+  const handleInputChange = (value: any, path: string[]) => {
+    const newFunctionInput = { ...functionInput };
```

Comment:
this sounds really we weird. `newFunctionInput` now equals `{ ...functionInput }` and we do a `setFunctionInput` right bellow. Do we still need this?